wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -108,6 +108,7 @@ export class SDKv2Route {
     const fromContext = await this.getV2ChainContext(fromChain);
     const toContext = await this.getV2ChainContext(toChain);
 
+    // TODO this is wrong... it should be filtering the output tokens...
```

Comment:
doesnt need to be investigated right now but I think this code is weird. would like to discuss with KP